### ISSUE TITLE
Gulp script requires 2 saves to load SCSS changes

### ISSUE THREAD

**Issue by maxmilton** [maxmilton/MaxMilton.com-Old-v3#1]
After running browser-sync via `gulp` when it detects scss cahnges you need to save twice. Once to generate the SCSS > CSS and the again to copy via Hugo.

This is due to no easy way to run hugo AFTER the css task. Gulp v4 allows tasks to run sequenchually, but Gulp v4 (which is currently being used) doesn't and runs everything in parallel.

I could fix this with a heap of aditional complexity to the gulpfile.js but I'd rather keep it as simple as possible for now.

**Comment by maxmilton** [comment]
Close as it's no longer relevant. Next version of the site will use a completely different, modernised approach.
